compiler/noirc_evaluator/src/ssa/opt/inline_functions_with_at_most_one_instruction.rs

```diff
@@ -25,18 +25,19 @@ impl Ssa {
 
             let entry_block_id = callee.entry_block();
             let entry_block = &callee.dfg[entry_block_id];
+            let instructions = entry_block.instructions();
 
             // Only inline functions with a single block
             if entry_block.successors().next().is_some() {
                 return false;
             }
 
             // Only inline functions with 0 or 1 instructions
-            if entry_block.instructions().len() > 1 {
+            if instructions.len() > 1 {
                 return false;
             }
 
-            let instructions = callee.dfg[entry_block_id].instructions();
```

Comment:
Is this a fix, separate from the tests?

Reply:
It's not fixing anything, just a small refactoring because these instructions have been accessed in a couple of ways.